source/Octopus.Server.Client/Repositories/Async/BasicRepository.cs

```diff
@@ -102,21 +103,36 @@ protected async Task EnsureServerIsMinimumVersion(SemanticVersion requiredVersio
         }
 
         public virtual async Task<TResource> Create(TResource resource, object pathParameters = null)
+        {
+            return await Create(resource, pathParameters, CancellationToken.None);
+        }
+        
+        public virtual async Task<TResource> Create(TResource resource, CancellationToken cancellationToken)
+        {
+            return await Create(resource, null, cancellationToken);
+        }
+
+        public virtual async Task<TResource> Create(TResource resource,  object pathParameters, CancellationToken cancellationToken)
```

Comment:
For nullability, should this be `object? pathParameters`?

Reply:
This is actually implementing `ICreate<TResource>.Create(TResource, object, CancellationToken`. 
Updating to on the interface `object?` is causing compiler error `CS8632`.

Reply:
Fair call 👍 and agreed. Looking at the scope of things, nullability looks like it should wait until a later pass.